test/resources/microshift-process.resource

```diff
@@ -112,3 +119,12 @@ Cleanup MicroShift
     Should Be Equal As Integers    ${rc}    0
 
     Enable MicroShift
+
+Delete Core Components
+    [Documentation]    Delete core components pods from MicroShift's control
+    ...    plane.
+    Run With Kubeconfig    oc delete deployment topolvm-controller -n openshift-storage
+    Run With Kubeconfig    oc delete daemonset topolvm-node -n openshift-storage
+    Run With Kubeconfig    oc delete deployment service-ca -n openshift-service-ca
+    Run With Kubeconfig    oc wait --for delete pod --selector=app.kubernetes.io/name=topolvm-csi-driver -n openshift-storage --timeout=120s
+    Run With Kubeconfig    oc wait --for delete pod --selector=app=service-ca -n openshift-service-ca --timeout=120s
```

Comment:
What heppens if pods are deleted before we get into the way command?

Reply:
The delete commands for daemonset/deployment return in milliseconds, plus the wait command returns immediately if there is no match for the selector, meaning the pods have already been deleted.